bluebrain/repo-bluebrain/packages/libsonata-report/package.py

```diff
@@ -41,6 +42,7 @@ def cmake_args(self):
             "-DSONATA_REPORT_ENABLE_SUBMODULES=OFF",
             "-DSONATA_REPORT_ENABLE_TEST=OFF",
             "-DSONATA_REPORT_ENABLE_WARNING_AS_ERROR=OFF",
+            "-DSONATA_REPORT_ENABLE_CONVERTER=ON",
```

Comment:
`nitpick`: Ideally you want to add this CMake option with a version check. But it won't be an error but a warring about unused camke var.